docs/modules/ROOT/pages/guide-keep3r.adoc

```diff
@@ -47,37 +49,54 @@ image::guide-keep3r-relayer-balance.png[Checking the balance of your Relayer]
 
-NOTE: Most jobs in the network do not require any bonded tokens to be executed, but you can modify the script by changing `collateralAmount` to bond some tokens if you want. If you do so, remember to transfer the tokens to be bonded to the Relayer address, or the Autotask will fail.
+NOTE: You can at any time choose to _Bond to Keep3r Network_ again. This will allow you to bond additional KP3R tokens, but will reset the waiting period back to 3 days.
 
-After bonding, there is a three-day delay enforced by the network before you can start using your Keeper. You can check your Autotask runs logs to see how much time is remaining. Once that time has passed, the script will take care of *activating* your Relayer as a Keeper.
+## Activate your Relayer
+
+After the 3-day waiting period is over, you will have the option to _Activate as keep3r_. Doing so will send a transaction to activate your Relayer as a Keeper in the network.
+
+image::guide-keep3r-activated.png[Relayer is now a Keeper]
+
+## Create an Autotask
 
-NOTE: Once your Relayer is activated, there is no need for it to go through the same bonding and activation process again if you modify the Autotask that executes the jobs, or add another Autotask connected to it.
+Now that you have your Relayer active as a Keeper, let's set up the Defender Autotask that will instruct the Relay to send them. An Autotask is a snippet of code that runs at regular intervals, in this case, we'll use it to automate the process of bonding, activating, searching for jobs, and executing them.
+
+Create a new Autotask in the xref:autotasks.adoc[Autotasks] section of Defender, choosing to run every one minute, and connecting it to the Relayer you had created previously.
+
+As for the code, paste https://github.com/OpenZeppelin/defender-autotask-examples/blob/master/keep3rs/src/multiple-jobs-keeper.js[this code snippet] from the `defender-autotask-examples` repository. Once you hit save, Defender will start executing the Autotask every minute, taking your Relayer through the bonding and activation process, and then executing available jobs.
 
 ## Executing Jobs
 
-Right after your Keeper is activated, it will start executing available jobs. The script is set up to look for available jobs in the `YearnV1EarnKeep3r`, `HegicPoolKeep3r`, and `UniswapV2SlidingOracle` contracts. You can check out your Autotask logs to see what your Keeper has done on each run.
+The autotask you have set up will automatically start looking for available jobs in the `YearnV1EarnKeep3r`, `HegicPoolKeep3r`, and `UniswapV2SlidingOracle` contracts. You can check out your Autotask logs to see what your Keeper has done on each run, and monitor your rewards on the Relayer page.
 
 WARNING: Keep in mind that sending the transaction does not automatically mean you will get rewarded for it: other Keepers may also be monitoring the job for work, and if their transaction gets mined before yours, you will not get the reward!
 
-## Coming soon...
```

Comment:
Is there any more functionality coming soon?

Reply:
Just the Defender ethereum provider